user-journeys/application_developer-foundational.html

```diff
@@ -16,80 +16,112 @@
+<br><br>
+        We start with hands-on tasks -- how to install and configure a simple Kubernetes cluster on your local machine, and how to deploy and manage applications on your new cluster. If you'd rather read more about Kubernetes terms and concepts first, you can skip to <a href="#3">Understand Kubernetes basics</a>.
+<br><br>
+        TODO: Add link back to setup from basics, depending on how much text we wind up with and what other navigation is available on the page
+<br><br>
+        TODO: decide how much intro conceptual material to put here. Working from <a href="https://kubernetes.io/docs/tutorials/kubernetes-basics/cluster-intro/">Using Minikube to Create a Cluster</a> here; see Kubernetes basics section for a different way of presenting the same information. TO DISCUSS: Large amount of near-duplicate material in different sections of the docs. Edit and move here? Edit in place and link? How much do we want to normalize/reconcile and remove?
+<br><br>
+        Kubernetes coordinates a highly available cluster of computers that are connected to work as a single unit. This cluster runs your application as a set of containers, which lets you decouple your application deployments from specific hosts. Kubernetes automates the distribution and scheduling of application containers across a cluster.
 </div>
     </div>
 
       <div class="topheader" id="1">
-      <span class="numberCircle"><span>1</span></span></i>&nbsp;&nbsp;Setup a development environment
+      <span class="numberCircle"><span>1</span></span></i>&nbsp;&nbsp;Set up Kubernetes
       </div>
 
       <div class="docsection1">
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean id felis non enim faucibus lacinia. Aliquam massa mauris, interdum a ex ut, sagittis rutrum nulla. In pellentesque est at molestie fringilla. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec eu mi at velit lacinia venenatis ac nec sem. In volutpat pellentesque dui ut commodo. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Duis sollicitudin eleifend felis non facilisis. Pellentesque leo urna, congue id auctor non, varius a nunc. Duis ultrices, odio ut hendrerit suscipit, nisi mauris dignissim mauris, nec bibendum ante neque ut augue. Interdum et malesuada fames ac ante ipsum primis in faucibus. Nam condimentum libero sit amet rutrum fermentum. Proin posuere condimentum odio. Aenean volutpat, ex vel tincidunt tincidunt, massa odio condimentum lectus, vel iaculis libero dolor in nisi.
+        To set up and explore a Kubernetes cluster for the first time, you can create a simple cluster locally with Minikube. Minikube gives you a cluster with a single node and all core Kubernetes components. You can set up a Minikube cluster just to explore, or, depending on your application, Minikube can also provide a useful development environment. (TODO: provide more detail here?)
```

Comment:
As we discussed, I think it would be nice to link to the interactive tutorial as a place to explore.